python/phylanx/ast/transformation.py

```diff
@@ -394,15 +483,14 @@ def _If(self, a, allowreturn=False):
         return s
 
     def _List(self, a, allowreturn=False):
-        symbol_info = full_node_name(a)
-        ret = "make_list%s(" % symbol_info
+        ret = "["
```

Comment:
Please use `make_list()` in the generated PhySL code. `[...]` will be interpreted as a (numeric) array (see: https://github.com/STEllAR-GROUP/phylanx/blob/master/phylanx/ast/parser/expression_def.hpp#L129-L131).